### DISCUSSION TITLE
Missing Tokens, quite a few!!!

### DISCUSSION THREAD

**Discussion by ghost** [blockscout/blockscout#9802] [Support]
I did some trades for Degen Pepe and other tokens yesterday, and I have the Transaction Records. I was pretty excited.

I used a brand new Metamask Account for this inside my MetaMask wallet. I used ETH>Degen>Tokens-- all good.  
Anyway, long story short, I have NO Degen Tokens in my Metamask Wallet? I do see a few Eth left, but no Degen, I also notice the logo for Degen network is a bit lame, not the purple one? Have I been hacked?

I do not see any transactions 'taking' the Tokens for the Account, so I was wondering if some kind hearted soul could possibly help me out!  Stuck. Thanks.

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Reply by ghost** [reply]
ok, fair enough, thanks

**Comment by ghost** [comment]
This matter was resolved with at Happy ending. 👍
